Detectors/MUON/MCH/Conditions/src/DCSNamer.cxx

```diff
@@ -0,0 +1,415 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "MCHConditions/DCSNamer.h"
+
+#include <array>
+#include <fmt/printf.h>
+#include <iostream>
+#include <set>
+
+namespace
+{
+const uint8_t invalidGroup{99};
+
+std::array<int, 156> detElemIds = {
```

Comment:
Well, let's call it a "controlled repetition"  ;-) The idea is to avoid to introduce, just for this single array, a dependency between libraries that are otherwise unrelated (or at least can be used separately). For instance I clearly don't want the transformation part to depend on (electronic) mapping.